denonavr/denonavr.py

```diff
@@ -394,6 +396,18 @@ def send_post_command(self, command, body):
                 "end point %s"), self._host, res.status_code, command)
             return False
 
+    def send_post_ir(self, command):
+        """Send IR command via HTTP post to receiver."""
+        res = requests.post("http://{host}:{port}{uri}{command}".format(
```

Comment:
Could you change this to a GET call?
I know it does not sound right, but I'm getting a _Read timed out_ error on HTTP POST, though the command was sent successfully. Using a GET request the command is still working, but there is not _Read timed out_ error